plugins/de.cau.cs.kieler.klighd.lsp/src/de/cau/cs/kieler/klighd/lsp/utils/KGraphMappingUtil.xtend

```diff
@@ -104,8 +106,21 @@ class KGraphMappingUtil {
         skedge.junctionPoints = new KVectorChain()
         skedge.junctionPoints.addAllAsCopies(0,kedge.getProperty(CoreOptions.JUNCTION_POINTS))
         skedge.junctionPoints.offset(new KVector(leftInset, topInset))
-        // Copy render scale.
-//        skedge.renderScale = kedge.getProperty(CoreOptions.TOP_DOWN_LAYOUT_RENDER_SCALE);
+
+        // TODO: not sure whether the above specially handled junction/routing points can be handled with the 
+        //       new property copying mechanism
+        // map all properties excepts those that are blacklisted
+        var properties = kedge.allProperties;
+        var blackList = KlighdDataManager.instance.blacklistedProperties;
+
+        for (propertyKVPair : properties.entrySet()) {
+            if (!containsPropertyWithId(blackList, propertyKVPair.key.id)) {
+                // TODO: remove this check once https://github.com/kieler/semantics/pull/13 has been merged
```

Comment:
I guess we want to remove this before the next release and not after this was merged. (also see cases below)

Reply:
Also in EObjectSerializer